.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -12,45 +12,56 @@
 1. - [ ] Retrieve the final run of the [Stage-DotNet](https://dev.azure.com/dnceng/internal/_build?definitionId=792&_a=summary) pipeline from internal release communications.
-1. - [ ] Run the [source-build-pre-release](https://dev.azure.com/dnceng/internal/_build?definitionId=1188) pipeline. When staging the pipeline run, click "Resources" and select the final run of Stage-DotNet mentioned above.
-     - This pipeline automatically uploads the dotnet source tarball to dotnetclimsrc and automatically pushes a branch to dnceng/security-partners-dotnet with the tarball contents.
-     - The `dotnet/installer` commit that represents the release will be in the logs for "Read Release Info"
-     - The `dotnet-installer-official-ci` and `dotnet-installer-source-build-tarball-build` pipeline will be linked in the logs for "Get Associated Pipeline Run IDs"
-        - [ ] Ensure the PoisonTests and SdkContentTests are passing.  Warnings indicate a baseline diff and should be inspected carefully.
-1. - [ ] Gather smoke-test prereqs ([automation tracking issue](https://github.com/dotnet/source-build/issues/3068))
-     - [ ] Retrieve smoke-test prereqs artifact for each architecture (e.g. x64 and arm64) from [tarball build](https://dev.azure.com/dnceng/internal/_build?definitionId=1011) (internal link)
-       - [ ] x64 - `Build Tarball CentOS7-Offline_Artifacts/dotnet-smoke-test-prereqs.6.0.xxx.tar.gz`
-       - [ ] arm64 - `Build Tarball Debian9-Offline_Artifacts/dotnet-smoke-test-prereqs.6.0.xxx.tar.gz`
-     - [ ] Retrieve additional packages from internal MSFT feed using [this project](https://gist.github.com/lbussell/47a3953686c218ede865e305478df74a).
-     - [ ] Create a new tarball of unique packages using [this script](https://gist.github.com/lbussell/5789974491e3d3ed737aac0e8b97b594).
-     - [ ] Upload smoke-test-prereqs tarball to dotnetclimsrc storage account, following the pattern of previous releases for directory and filename.
-         - Never overwrite a tarball. At least change the blob storage virtual dir to represent a new build. This can help avoid timing issues and make it more obvious if stale links were accidentally re-sent rather than new ones.
-1. - [ ] Update dotnet-security-partners
-     - [ ] Submit a PR to the appropriate `release/*` branch on [`dnceng/security-partners-dotnet`](https://dev.azure.com/dnceng/internal/_git/security-partners-dotnet). A branch was automatically created with the source-build-pre-release pipeline, you just have to submit the PR. ([automation tracking issue for this step](https://github.com/dotnet/source-build/issues/3069))
-     - [ ] squash merge the PR once CI finishes successfully.
+1. - [ ] Run the [source-build-pre-release](https://dev.azure.com/dnceng/internal/_build?definitionId=1188) pipeline.
+    - ⚠️ 6.0 / 7.0: [source-build-pre-release](https://dev.azure.com/dnceng/internal/_build?definitionId=1188)
```

Comment:
You mention the possibly of unifying these pipelines.  If the remain separate, I think the 6.0/7.0 pipeline should be renamed for better clarity.  e.g. `source-build-pre-release-6.0-7.0`, `source-build-pre-release-pre-8.0` or `source-build-pre-release-pre-VMR`